src/encoded/tests/test_audit_genetic_modification.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+
+@pytest.fixture
+def genetic_modification(testapp, lab, award):
+    item = {
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'modified_site_by_coordinates': {
+            'assembly': 'GRCh38',
+            'chromosome': '11',
+            'start': 20000,
+            'end': 21000
+        },
+        'purpose': 'repression',
+        'category': 'deletion',
+        'method': 'CRISPR',
+        'zygosity': 'homozygous'
+    }
+    return testapp.post_json('/genetic_modification', item).json['@graph'][0]
+ 
+
+def test_genetic_modification_reagents(testapp, genetic_modification, source):
+    res = testapp.get(genetic_modification['@id'] + '@@index-data')
+    errors = res.json['audit']
+    errors_list = []
+    for error_type in errors:
```

Comment:
What about `errors_list = [error for v in errors.values() for error in v]` instead of 
```
errors_list = []
    for error_type in errors:
        errors_list.extend(errors[error_type])

```

Reply:
pushed an updated version